controllers/apply/awards-for-all/fields.js

```diff
@@ -1019,6 +1046,65 @@ module.exports = function fieldsFor({ locale, data = {} }) {
         };
     }
 
+    function fieldProjectTotalCosts() {
+        const budgetTotal = sumBy(
```

Comment:
We can compute the project total here allowing us to do the min budget check using a standard `min` validation rather than needing a custom extension.